index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Additional 2</title>
+  <link rel="stylesheet" href="./Styles/style.css">
+</head>
+<body>
+  <div class="block">
+    <div class="red"></div>
```

Comment:
add to every div another class block
`<div class=" block red"></div>`
`<div class=" block green"></div>` and etc